Wikipedia/Code/ArticleViewController.swift

```diff
@@ -1045,6 +1045,14 @@ private extension ArticleViewController {
         if let altTextExperimentViewModel {
             self.navigationItem.titleView = nil
             self.title = altTextExperimentViewModel.localizedStrings.articleNavigationBarTitle
+
+            let rightBarButtonItem = UIBarButtonItem(
+                image: UIImage(systemName: "ellipsis.circle"),
```

Comment:
We still have several image literals all over the code, but we're trying to move on to using our WMFSFSymbolIcon component for consistency.

So this bit could be:

`UIBarButtonItem(image: WMFSFSymbolIcon.for(symbol: .ellipsisBubble)`